lib/activeadmin_addons/active_admin_config.rb

```diff
@@ -3,6 +3,7 @@ class ActiveAdmin::Views::Pages::Base
 
   def build(*args)
     original_build(args)
+    body = find_by_tag("body").first
```

Comment:
Use find_by instead of dynamic find_by_tag.

Reply:
I wish I could, but `Arbre::Element` only provides `find_by_tag` which is an alias for `get_elements_by_tag_name`
https://github.com/activeadmin/arbre/blob/master/lib/arbre/element.rb#L106